sdk/identity/identity/src/credentials/visualStudioCodeCredential.ts

```diff
@@ -20,6 +24,39 @@ const AzureAccountClientId = "aebc6443-996d-45c2-90f0-388ff96faa56"; // VSC: 'ae
 const VSCodeUserName = "VS Code Azure";
 const logger = credentialLogger("VisualStudioCodeCredential");
 
+/**
+ * Attempts to load the tenant from the VSCode configurations of the current OS.
+ * If it fails at any point, returns undefined.
+ */
+export function getTenantIdFromVSCode(): string | undefined {
+  const commonSettingsPath = ["Code", "User", "settings.json"];
```

Comment:
Some people use VS Code Insiders, should we check in that configuration folder too?  If they have both editions installed, which one wins?

Reply:
I asked the same for the identity team, but they are only checking at this folder in other languages, so I don't know where else to look for references. I more or less know how the VSCode internals are finding these configurations, but it's not a cheap process, since they get aggregated from several places into memory, and we can't have their modules as dependencies.

If we want to support insiders, what would be the folders we should look for?
Any help appreciated.

Reply:
If the Identity team is only checking in the stable VS Code folder, I think it's fine for us to do so as well.